.github/workflows/matrix.yml

```diff
@@ -119,7 +119,6 @@ jobs:
     steps:
     - name: install dependencies
       run: |
-        brew uninstall --ignore-dependencies --force pkg-config@0.29.2
         brew install meson usb.ids gobject-introspection libarchive json-glib protobuf-c vala gi-docgen
```

Comment:
don't you need `[brew install] readline` here too?

Reply:
It's already installed according to my earlier attempts